apps/frontend/src/components/global/sidebaritems/SidebarFileList.vue

```diff
@@ -94,7 +116,8 @@ export default class SidebarFileList extends mixins(ServerMixin, RouteMixin) {
     return typeof this.file?.database_id !== 'undefined' || this.saving;
   }
 
-  save_to_database(file: EvaluationFile | ProfileFile) {
+  save_to_database(file: EvaluationFile | ProfileFile | ChecklistFile) {
```

Comment:
expectation check: when a checklist file has already been saved to the db, and the user makes changes to the checklist within the editor, what should occur?

a) those changes remain purely on local side until the user hits the save button again from the sidebar to overwrite the version of the file in the db

b) those changes are autosaved and so are continuously overwriting the db